readme.md

```diff
@@ -319,6 +319,7 @@
 - [Litekart](https://www.litekart.in)
 - [NuxtJS Documentation](https://nuxtjs.org)
 - [Vaggelis Yfantis](https://octoper.me)
+- [thismonthrocks](https://thismonth.rocks)
```

Comment:
Can you format the name better like maybe `This Month Rocks`

Reply:
Fixed.